Tests/EventListener/BodyListenerTest.php

```diff
@@ -76,7 +76,7 @@ public function testOnKernelRequest($decode, Request $request, $method, $expecte
         $this->assertEquals($request->request->all(), $expectedParameters);
     }
 
-    public static function testOnKernelRequestDataProvider()
+    public static function onKernelRequestDataProvider()
```

Comment:
renamed some data providers so that they are not executed as tests (which makes them being reported as risky)